### DISCUSSION TITLE
Treat as motion sensor instead of occupancy sensor

### DISCUSSION THREAD

**Discussion by onegrizzle0g** [itavero/homebridge-z2m#742] [Q&A] [answered]
Hi there,
Does anyone know how to force zigbee motion sensors to show as motion sensors instead of occupancy sensors in HomeKit?

**Comment by itavero** [comment, marked as answer]
You can use a device specific configuration for that. Please refer to:
- [generic example config here, so you can see where to put it](https://z2m.dev/config.html)
- [example at bottom of this page, specific for occupancy sensors](https://z2m.dev/sensors.html)
